schema/CbsdRecordData.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Registration parameters provided for a CBSD",
+  "type": "object",
+  "required": [ "fccId", "cbsdCategory", "airInterface", "installationParam", "measCapability"],
+  "properties": {
+    "userId": { "type": "string",
+                "description": "Unique user designator for the operator. Generation mechanism TBD" },
+    "fccId": { "type": "string",
+               "description": "The certification FCC ID for the hardware" },
+    "cbsdSerialNumber": { "type": "string",
+                          "description": "Unique identifier for the CBSD chosen by the manufacturer" },
+    "cbsdCategory": { "enum" : [ "A", "B" ],
+                      "description": "A if the device is Category A; B if it is Category B" },
+    "callSign": { "type": "string",
+                  "description": "The call sign for the operator. Meaning and origin TBD" },
+    "measCapability": { "type": "array",
+                        "description": "List of supported sensing capabilities of the device",
+                        "items": { "enum": [ "RECEIVED_POWER_WITHOUT_GRANT", "RECEIVED_POWER_WITH_GRANT" ],
+                                   "description": "Measurement capabilities of the CBSD." }
+                      },
+    "groupingParam": {
+      "type": "array",
+      "description": "List of group memberships of this CBSD.",
+      "items": { "$ref": "#/groupParamObject" }
+    },
+    "cbsdInfo": { "$ref": "#/cbsdInfoObject" },
+    "airInterface": { "$ref": "#/airInterfaceObject" },
+    "installationParam": { "$ref": "file:InstallationParamData.schema.json" }
+  },
```

Comment:
Need to add "cpiSignatureData"?

Reply:
as I understood, according to an email thread we raised with WG4 and SAS admins, for the CBSD record in the dump, SAS doesn't need to add the cpiSignatureData, and if it receives the installation param in cpiSignatureData, SAS should extract them and put them in the dump without encoding them.

Reply:
Sounds good.